$OpenBSD: patch-docs_polkit_Makefile_am,v 1.8 2014/03/26 14:56:23 ajacoutot Exp $

Remove when updating to a newer release with integrated bsd_auth(3) support.

--- docs/polkit/Makefile.am.orig	Wed Mar 26 15:38:54 2014
+++ docs/polkit/Makefile.am	Wed Mar 26 15:39:16 2014
@@ -80,11 +80,7 @@ HTML_IMAGES = 						\
 # Extra options to supply to gtkdoc-fixref
 FIXXREF_OPTIONS=
 
-if ENABLE_GTK_DOC
-include $(top_srcdir)/gtk-doc.make
-else
 CLEANFILES =
-endif
 
 CLEANFILES +=   *~				\
 		polkit-1-scan.*			\
